imageregistry/v1/types.go

```diff
@@ -301,6 +302,7 @@ type ImageRegistryConfigRequestsLimits struct {
 	// maxWaitInQueue sets the maximum time a request can wait in the queue
 	// before being rejected.
 	// +optional
+	// +kubebuilder:validation:Format=duration
```

Comment:
@ricardomaraschini is this format documented somewhere?

Reply:
After some digging around I managed to find out where this validation is made[1]. 

[1] https://github.com/go-openapi/strfmt/blob/master/duration.go

Reply:
Documented here https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.18/#jsonschemaprops-v1-apiextensions-k8s-io